.github/workflows/linting.yml

```
@@ -0,0 +1,17 @@
+name: Linting
+
+on:
+  push:
```

Recommend scoping push to main so it doesn't run twice on PRs.
```suggestion
  push:
    branches: [main]
```